src/content/posts/2023-10-24-git-workflow.md

```diff
@@ -0,0 +1,132 @@
+---
+title: "Tips to git good with git"
+slug: git-workflow
+date: 2023-10-24 12:00:00
+tags:
+- git
+categories:
+- git
+twitter_text: "Some helpful tips and commands to manage your git workflow."
+authors: 
+- Jaime Jones
+image: https://images.unsplash.com/photo-1604918598683-28d4e6e2986d?auto=format&fit=crop&q=80&w=3570&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D
+image_url: https://unsplash.com/photos/black-bare-tree-under-white-sky-during-daytime-Ku-1SYS0o7k
+image_credit: Mila Tovar
+---
+
+Whether you love it or hate it, it's undeniable that git (or your source control of choice, but this post is about git, so git on outta here if you use something else) is an integral part of our lives as developers. There are a lot of ways to manage respositories and codebases, but the [forking workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/forking-workflow) is the workflow that has worked for me and for our team with only minimal nuclear level resets required. Who here has had to delete and refork a repository and then clone it down fresh again? If that's been you (don't worry, it's probably been all of us at some point), maybe this post can help.
+
+### Managing repositories
```

Comment:
should this be
```suggestion
## Managing repositories
```

Reply:
I get that this is semantic, but they're big and ugly

Reply:
![fine](https://media0.giphy.com/media/Id1Ay5MD5IsF6Or51v/giphy.gif)